pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -210,6 +214,7 @@ func (f *framework) RunPreFilterPlugins(
 			klog.Error(msg)
 			return NewStatus(Error, msg)
 		}
+		klog.V(10).Infof("Running %q prefilter plugin for pod %q: %v", pl.Name(), pod.Name, status.Message())
```

Comment:
nit: Consider moving this before the `if !status.IsSuccess()`, otherwise the status will always be success here. Same comment for other occurrences.

Reply:
In fact, this is to take into account that will print information duplicates with `klog.Error(msg)`.